### ISSUE TITLE
Dark high contrast theme has dark text on dark background

### ISSUE THREAD

**Issue by ChristiaanvdMeer** [microsoft/vscode#213457]
Type: <b>Bug</b>

1. Set theme to "Dark High Contrast"
2. Go to Editor Playground > Formatting
3. Observe several dark texts on dark background where Keyboard Shortcuts should be.
4. Optional: search for other places with dark text on dark background in this theme.

![image](https://github.com/microsoft/vscode/assets/136791941/541f0ebf-50ea-4fec-8902-85b999597af9)

VS Code version: Code 1.89.1 (dc96b837cf6bb4af9cd736aa3af08cf8279f7685, 2024-05-07T05:13:33.891Z)
OS version: Windows_NT x64 10.0.22631
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-10300H CPU @ 2.50GHz (8 x 2496)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: enabled_on<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_graphite: disabled_off<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled|
|Load (avg)|undefined|
|Memory (System)|7.83GB (1.99GB free)|
|Process Argv|--crash-reporter-id a60066b5-051e-447c-abe9-d0ddd16c2e37|
|Screen Reader|no|
|VM|0%|
</details>Extensions: none<details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vspor879:30202332
vspor708:30202333
vspor363:30204092
vswsl492cf:30256860
vscoreces:30445986
vscod805cf:30301675
binariesv615:30325510
vsaa593cf:30376535
py29gd2263:31024239
c4g48928:30535728
azure-dev_surveyone:30548225
962ge761:30959799
pythongtdpath:30769146
welcomedialog:30910333
pythonidxpt:30866567
pythonnoceb:30805159
asynctok:30898717
pythontestfixt:30902429
pythonregdiag2:30936856
pythonmypyd1:30879173
pythoncet0:30885854
2e7ec940:31000449
pythontbext0:30879054
accentitlementst:30995554
dsvsc016:30899300
dsvsc017:30899301
dsvsc018:30899302
cppperfnew:31000557
dsvsc020:30976470
pythonait:31006305
chatpanelt:31048053
dsvsc021:30996838
bdiig495:31013172
pythoncenvptcf:31049071
a69g1124:31058053
pythonprt:31056678
dwnewjupyter:31046869
26j00206:31048877

```

</details>
